packages/smooth_app/lib/helpers/picture_capture_helper.dart

```diff
@@ -20,25 +16,11 @@ Future<bool> uploadCapturedPicture(
 }) async {
   final AppLocalizations appLocalizations = AppLocalizations.of(context);
   final LocalDatabase localDatabase = context.read<LocalDatabase>();
-  final String uniqueId = _getUniqueId(imageField, barcode);
-  final BackgroundImageInputData backgroundImageInputData =
-      BackgroundImageInputData(
-    processName: IMAGE_UPLOAD_TASK,
-    uniqueId: uniqueId,
-    barcode: barcode,
-    imageField: imageField.value,
-    imagePath: File(imageUri.path).path,
-    languageCode: ProductQuery.getLanguage().code,
-    user: jsonEncode(ProductQuery.getUser().toJson()),
-    country: ProductQuery.getCountry()!.iso2Code,
+  await BackgroundTaskImage.addTask(
+    barcode,
+    imageField: imageField,
+    imageFile: File(imageUri.path),
   );
-  await TaskManager().addTask(
-    Task(
-      data: backgroundImageInputData.toJson(),
-      uniqueId: uniqueId,
-    ),
-  );
-
   localDatabase.notifyListeners();
   // ignore: use_build_context_synchronously
   ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
Can we somehow get rid of this //ignore, I tried but couldn't
Just asking!

Reply:
No we can't.
We could if we were inside a `Widget`, where we can test `if(!mounted)` instead.
That would be for another refactor PR.